doc/source/user_guide/window.rst

```diff
@@ -76,6 +76,27 @@ which will first group the data by the specified keys and then perform a windowi
     to compute the rolling sums to preserve accuracy as much as possible.
 
 
+.. versionadded:: 1.3
+
+Some windowing operations also support the ``method='table'`` option in the constructor which
+performs the windowing operaion over an entire :class:`DataFrame` instead of a single column or row at a time.
+This can provide a useful performance benefit for :class:`DataFrame`s with a many columns or rows
+(with the corresponding ``axis`` argument) or the ability to utilize other columns during the windowing
+operation. The ``method='table'`` option can only be used if ``engine='numba'`` is specified
+in the corresponding method call.
+
+
+.. ipython:: python
+
+   def weighted_mean(x):
```

Comment:
add: for example a weighted mean calculation, link to: https://en.wikipedia.org/wiki/Weighted_arithmetic_mean